go/vt/discovery/healthcheck.go

```diff
@@ -67,6 +69,8 @@ var (
 	hcErrorCounters          = stats.NewMultiCounters("HealthcheckErrors", []string{"Keyspace", "ShardName", "TabletType"})
 	hcMasterPromotedCounters = stats.NewMultiCounters("HealthcheckMasterPromoted", []string{"Keyspace", "ShardName"})
 	healthcheckOnce          sync.Once
+	tabletURLTemplateString  = flag.String("tablet_url_template", "http://{{.GetTabletHostPort}}", "format string describing debug tablet url formatting.")
```

Comment:
We (at Slack) have a similar issue and have never put the time into addressing it so thanks for this feature.

However I'd like to be able to format an URL based on just the hostname without the port. From my brief understanding of the code it seems like I could do this with `"http://{{.GetHostNameLevel 0}}.some.domain"` -- if so that would be perfect, but if not then it'd be great to enable this as part of this change.

Reply:
GetHostNameLevel is there to help divide up the hostname into it's components. If you want the whole thing I think `http://{{.Tablet.Hostname}}` should work.

Reply:
I actually just tested this out and it indeed works.

Reply:
I feel like this is a non-trivial feature. How about leaving a pointer in this help text for interested users?

e.g. add: "See the Go code for getTabletDebugURL() how to customize this."

And then you can document the feature and tricks like `{{.Tablet.Hostname}}` there.